lib/src/components/NxNexusPageHeader/NxNexusPageHeader.scss

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-container-helpers';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-page-header--nexus {
+  .nx-page-header__inner {
+    padding-left: 0;
+  }
+
+  .nx-product {
+    display: -ms-grid;
+    display: grid;
+    -ms-grid-columns: 72px 16px 1fr;
+    grid-template-columns: 72px 1fr;
+    -ms-grid-rows: 39px 0 auto;
```

Comment:
What's the purpose of the fake middle row here?

Reply:
> What's the purpose of the fake middle row here?

Because IE11 doesn't respect `gap` this adds spacing

Reply:
> Because IE11 doesn't respect gap this adds spacing

But it's 0.

Reply:
> > Because IE11 doesn't respect gap this adds spacing
> 
> But it's 0.

Dammit Autoprefixer! I didn't notice that: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/3adc468fc17c02341b1bfa6efdc5abaeea26e420

Reply:
Pretty sure more adjustments are necessary for this regarding `-ms-grid-row-span` and `-ms-grid-row` properties.

Reply:
> Pretty sure more adjustments are necessary for this regarding `-ms-grid-row-span` and `-ms-grid-row` properties.

<img width="642" alt="Screen Shot 2020-11-03 at 12 45 33 PM" src="https://user-images.githubusercontent.com/1991443/98021607-7f3d6600-1dd2-11eb-8a10-55f747786a65.png">
 Not sure what you mean, it looks good to me.

Reply:
> Not sure what you mean, it looks good to me.

Looking again, what you have is more correct than I had assumed.  I do think however you now have some `-ms-grid-row-span: 1` props which don't really do anything.

Reply:
> > Not sure what you mean, it looks good to me.
> 
> Looking again, what you have is more correct than I had assumed. I do think however you now have some `-ms-grid-row-span: 1` props which don't really do anything.

Ah, removed here: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/a59db605de748ca8ca676948aea63714c5540162